### PR TITLE
Database migration for Yarn Package Manager

### PR DESCRIPTION
**Pull request by nirzari**

### CODE REVIEW

File: cachito/web/migrations/versions/eff9db96576e_add_yarn.py
Comment (chmeliik) [inline cachito/web/migrations/versions/eff9db96576e_add_yarn.py:14]: `black` wants double quotes here, other than that LGTM

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Review by athos-ribeiro — APPROVED** [review]
LGTM

We should wait for the CI before merging

**Pull request merged**
